docs/env_variables.md

```diff
@@ -327,6 +331,11 @@ checks or alter some of the more exotic semantics of the tool:
     (`-i in`). This is an important feature to set when resuming a fuzzing
     session.
 
+  - `AFL_CODE_COVERAGE` will enable code coverage estimation. See also 
+    [coverage_estimation.md](./coverage_estimation.md). Related enviroment
+    variables: `AFL_ABUNDANT_CUT_OFF`, `AFL_CRASH_ON_HASH_COLLISION` and
+    `AFL_N_FUZZ_SIZE`
+
```

Comment:
hmm I would propose to move AFL_ABUNDANT_CUT_OFF and AFL_CRASH_ON_HASH_COLLISION just below here.